### DISCUSSION TITLE
RustDesk Recording Issue

### DISCUSSION THREAD

**Discussion by almedina-tech** [rustdesk/rustdesk#10257] [General]
Hello! is there somebody else who experienced a recording issue with RustDesk? How did you solve it? It never worked on my end even if it's enabled in the settings.

**Comment by rustdesk** [comment]
@21pages

**Comment by 21pages** [comment]
Could you try out https://github.com/21pages/rustdesk/releases/download/recording-build/rustdesk-1.3.5-x86_64.exe on the control(local) side? After recording, upload the log at `%AppData%\RustDesk\log\RustDesk_rCURRENT.log`, remember to remove the lines containing personal informations.

If you want to record incomming sessions, please use it in the controlled side,  please uninstall the old one first, the log path is:
 installed: `C:\Windows\ServiceProfiles\LocalService\AppData\Roaming\RustDesk\log\server\RustDesk_rCURRENT.log`
 not installed: `%AppData%\RustDesk\log\RustDesk_rCURRENT.log`

If not using windows, try out https://github.com/21pages/rustdesk/releases/tag/recording-build and https://github.com/rustdesk/rustdesk/wiki/FAQ#access-logs

**Comment by almedina-tech** [comment]
I already have the RustDesk and the Automatic record incoming session is enabled

![image](https://github.com/user-attachments/assets/a6830c11-0aaf-4900-8482-3340255a0a52)

But when I go to the path directory, it's empty.

**Reply by rustdesk** [reply]
Please follow https://github.com/rustdesk/rustdesk/discussions/10257#discussioncomment-11527437

**Comment by almedina-tech** [comment]
It fixed. Thank you!

**Reply by rustdesk** [reply]
https://github.com/21pages/rustdesk/releases/download/recording-build/rustdesk-1.3.5-x86_64.exe fixed you?

**Reply by 21pages** [reply]
It's the same as nightly. Maybe it's version before 1.2.6 when av1 record is not supported.
